duckduckgo/iOS
# Update Kingfisher to 7.0.0

Task/Issue URL: https://app.asana.com/0/414235014887631/1201162855622744/f
Tech Design URL:
CC:

**Description**:
This PR updates our Kingfisher dependency to version >=7.0.0, fixing errors while archiving with Xcode 13.

**Steps to test this PR**:
1. Test that functionality related to image caching works as expected; see the [Kingfisher changelog](https://github.com/onevcat/Kingfisher/blob/master/CHANGELOG.md) for more information on what's changed
2. Run the adhoc build script to verify that build succeeds without errors.
3. Ensure retained compliance according to https://app.asana.com/0/414709148257752/1200709505676144/f




**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
